archive.conf

```diff
@@ -11,8 +11,8 @@ ARCHIVE_USER=archive
 ARCHIVE_GROUP=archive
 
 # Package extensions
-PKGEXT='.pkg.tar.xz'
-PKGSIG="$PKGEXT.sig"
+PKGREGEX='\.pkg\.tar(\.(gz|bz2|xz|zst|lrz|lzo|Z|lz4|lz))?'
+PKGSIG="$PKGREGEX.sig"
```

Comment:
Dot should be escaped.

Reply:
Meh, don't think it's a big deal

Reply:
but this is a wip pull request, whats the deal with just doing the _right_ thing, and not something that just _also_ works for the correct file